### DISCUSSION TITLE
Is the project still actively maintained?

### DISCUSSION THREAD

**Discussion by chuanhhoang** [writefreely/writefreely#550] [Questions / Support]
Is this project still actively maintained? There aren't any commits for months. Bugs are not fixed ....

**Comment by carbontwelve** [comment]
I am pretty sure that it is, the commercial side of the project recently had an update https://write.as/updates/ and Matt has recently answered questions on https://discuss.write.as/c/writefreely/15

**Comment by thebaer** [comment]
Yes, the project hasn't gone anywhere!

But we need more people to contribute -- whether developing, reviewing code, writing documentation, triaging bugs, etc. -- if they want to see it evolve faster. I'm only one person doing all the work (providing support, taking feedback, writing code, funding this with the commercial version,...) so you can expect it to move at a one-person pace until more people step up to help.

As I [mentioned here](https://writing.exchange/@matt/108296096956116443), one issue is that we can't get a new release out right now, due to a bug in the tool we use to cross-compile WF: techknowlogick/xgo#155. If anyone wants to see concrete progress and can help out with this, that would be a great place to start!

**Reply by ghost** [reply]
I would be interested in helping out, I've been wanting to find an opensource project to contribute to for a long time now. I have -some- time. I'm quite comfortable in the areas of writing documentation, HTML, CSS. I have some novice+ level experience with JS and novice with GIT. I unfortunately do not have any experience with GO currently, but I'd be willing to put some time in to learn a bit. I'm not sure I'd be much help trying to sort out the compiling issue here though. I know you're busy, but if you or someone else here a little more experienced than me could point me in the right direction on some things that need taking care of, I'd be glad to try and chip and help out. I just recently, like a week ago, setup a writefreely instance for myself.

**Reply by carbontwelve** [reply]
I'd certainly like to see some movement on the project, I'll see if I can schedule in some time to take a look at the xgo issue you mention.

**Reply by jccpalmer** [reply]
I have experience as a technical writer, so while I can't contribute to code, I might be able to assist with documentation.

**Reply by chartier** [reply]
Hey Matt! We talked on Masto about me possibly helping with general user-facing comms. 

I have a lot of experience speaking to a variety of audiences across blog posts, documentation, copywriting, and social media, and especially translating developer-speak for regular folks. One thing I _can't_ help with is viral TikTok dances; gotta draw a line somewhere.

I started my career in tech news writing for TUAW, Ars Technica, and Macworld, then sidestepped to helping indie developers since I love the space. I've worked with 1Password, Omni Group, a variety of Indies, and currently the Bear notes app. I haven't contributed to projects like this before, but I'm a quick learn.

If any of this could be useful, let me know.
